src/lib.rs

```diff
@@ -410,7 +411,7 @@ pub fn validate_commit<H, N, S, I, C: Chain<H, N>>(
 			chain.is_equal_or_descendent_of(
 				commit.target_hash.clone(),
 				signed.precommit.target_hash.clone(),
-			)
+			).unwrap_or(false)
```

Comment:
@andresilva i always forget what the semantics of `Ok(None)` are for `validate_commit`. That means that it validated successfully, but the result was that the commit is bad?

Reply:
Yeah, this function should return the precommit ghost, which should exist if the commit is well-formed and hence the function returns `Ok(Some(_))` when given a valid commit. I think in this case we shouldn't call `unwrap_or(false)` and instead just do `?`.